config/freertos/esp32/README.md

```diff
@@ -4,3 +4,54 @@
 - You can run idf targets like `menuconfig` or `monitor` by specifing the target as an argument to the `build_firmware.sh` script
 - The GPIO pins for the configured serial port can be set with `menuconfig` (see `micro-ROS Transport Settings` menu)
 - ESP32 only runs in singlecore mode (`CONFIG_FREERTOS_UNICORE=y` setting)
+
+
+# Notes for ESP32-CAMERA 
+
+- SPIRAM is requierd only for ESP32-CAMERA and can cause a fatal error in ESP32 (Enable in  `menuconfig Component config > ESP32-specific` OR  add `CONFIG_ESP32_SPIRAM_SUPPORT=y` to `sdkconfig.defaults` file).
```

Comment:
Regarding this, is possible to explain it a README.md inside the `freertos_apps` app folder?